pan3d/__main__.py

```diff
@@ -0,0 +1,17 @@
+from argparse import ArgumentParser
+from dataset_builder import DatasetBuilder
+
+parser = ArgumentParser(
+    prog="Pan3D",
+    description="Launch the Pan3D Dataset Viewer / Builder",
+)
+
+parser.add_argument("-b", "--config_path")
+parser.add_argument("-d", "--dataset_path")
+
+args = parser.parse_args()
+
+viewer = DatasetBuilder(dataset_path=args.dataset_path)
+if args.config_path:
```

Comment:
builder...

Reply:
Ok, changed in [478af46](https://github.com/Kitware/pan3d/pull/28/commits/478af46b55a1d8e87be39ade88fadfcab4cee527)